.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -6,27 +6,39 @@ labels: bug
 
 ---
 
+**⚠️IMPORTANT⚠️ if you do not fill this out, we cannot help you.**
```

Comment:
Maybe make this even more forceful, something like "we will automatically close any reports which are not filled out properly"